### PR TITLE
Implement Book Duck

### PR DESCRIPTION
**Pull request by RokoVarano**
Book functionality has been implemented using redux.
- Books and author can be added using the form
- Books can be deleted using the 'Delete' Button in each book widget.

### CODE REVIEW

File: src/components/BookForm/BookForm.jsx [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+
+    // dispatch an action and pass it the newBook object (your action's payload)
+    dispatch(addBook(newBook));
+  };
+
+  return (
+    <div className="book-form">
+      <input type="text" id="book-input-title" placeholder="" />
+      <input type="text" id="book-input-author" placeholder="" />
+      <button type="button" onClick={() => submitBookToStore(document.getElementById('book-input-title').value, document.getElementById('book-input-author').value)}>Add Book</button>
```
Comment (bellom) [inline src/components/BookForm/BookForm.jsx]: Awesome job trying to get the values from the input tag.

But in React, it is not professional to use `document.getElementById`, the reason is that you may have multiple instances of the same React component on a page.

Let's look into a better way of doing this;

[How to Access DOM Nodes in React with Refs](https://www.newline.co/@dmitryrogozhny/how-to-access-dom-nodes-in-react-with-refs--57d97b3d)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @RokoVarano  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_1. Kindly check the comments under the review._

_2. To make your commit messages look more professional, you should always capitalize them._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm/BookForm.jsx (hunk 1 above)
Comment (RokoVarano) [reply src/components/BookForm/BookForm.jsx]: Not a requirement, but complied.

**Comment by Davidosky007** [comment]
## Hi @RokoVarano 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**